### PR TITLE
Add DeepDTA architecture in README and fix some issues in code

### PR DESCRIPTION
**Pull request by pz-white**
### Description
- add model architecture and results for DeepDTA README.
- Fix some small issues in pykale DeepDTA API.

### Status
Ready

### Types of changes
- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] In-line docstrings updated and documentation `docs` updated.

### CODE REVIEW

**Comment by haipinglu** [comment]
@pz-white Have you read the PR summary? The description is the template and the status is the description.

File: examples/bindingdb_deepdta/README.md [hunk 1]
```diff
@@ -41,3 +41,50 @@ python main.py --cfg configs/IC50-DeepDTA.yaml
 python main.py --cfg configs/Kd-DeepDTA.yaml
 python main.py --cfg configs/Ki-DeepDTA.yaml
 ```
+
+### 6. Results
+Here are the MSE loss results for the three BindingDB datasets, the minimal validation loss's epoch is saved as best
```
Comment (haipinglu) [inline examples/bindingdb_deepdta/README.md:46]: Two sentences are put together without a conjunction. Use a conjunction or break into two sentences.

Missing article "the" in front of "best ...": "the best ...

File: kale/evaluate/metrics.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+def c_index(y, y_pred):
+    """
+    Calculate Concordance Index (CI), which is a metric to measure proportion of concordant pairs between real and
```
Comment (haipinglu) [inline kale/evaluate/metrics.py:3]: - Insert "the" before "Concordance Index", before "proportion".
- Provide a hyperlink to concordant pairs (I do not know what it means), e.g. wiki is usually the first choice (if available) https://en.wikipedia.org/wiki/Concordant_pair

**Review by haipinglu — COMMENTED** [review]
My comments are all about writing (mainly English grammar). We can merge after fixing them.

**Comment by haipinglu** [comment]
@pz-white Before I automate them, please add label and project (again, these were reminders in the PR but you deleted them without action).

File: examples/bindingdb_deepdta/README.md (hunk 1 above)
Comment (pz-white) [reply examples/bindingdb_deepdta/README.md]: Resolved

File: kale/evaluate/metrics.py (hunk 2 above)
Comment (pz-white) [reply kale/evaluate/metrics.py]: Resolved and add kale.evaluate in documentation.

**Comment by pz-white** [comment]
@haipinglu Thanks for reviewing and comments, I have changed following the isses on comments.

**Review by haipinglu — APPROVED** [review]

**Pull request merged**